charts/grafana/values.yaml

```diff
@@ -435,11 +436,11 @@ enableServiceLinks: true
 
 ## Additional grafana server secret mounts
 # Defines additional mounts with secrets. Secrets must be manually created in the namespace.
-extraSecretMounts: []
-  # - name: secret-files
-  #   mountPath: /etc/secrets
-  #   secretName: grafana-secret-files
-  #   readOnly: true
+extraSecretMounts: 
+    - name: ge-license
```

Comment:
this secret must be manually created outside the helm chart. according to this:
```
## Additional grafana server secret mounts
# Defines additional mounts with secrets. Secrets must be manually created in the namespace.
```
So, not enterprise users won't have it and it will prevent the pod from starting up.  As long as I remember.

Reply:
Good point. So maybe we take that out.